webservices/resources/reports.py

```diff
@@ -114,20 +114,15 @@ def build_query(self, committee_type=None, **kwargs):
             default_schemas,
         )
         query = reports_class.query
-        # Eagerly load committees if applicable
         if hasattr(reports_class, 'committee'):
-            query = reports_class.query.join(reports_class.committee).options(sa.orm.joinedload(reports_class.committee))
```

Comment:
The previous code I had made nested and after looking at it after some time away from it I had no idea really what I was trying to do.  It was tough finding a query that could make the relation properly without breaking the tests.  I'm fairly confident this is the write approach now.